packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.java

```diff
@@ -51,8 +52,21 @@
     void onSuccess(AssetEntity assetEntity, boolean isNew);
   }
 
-  public static void downloadFileToPath(Request request, final File destination, final FileDownloadCallback callback) {
-    downloadData(request, new Callback() {
+  private static OkHttpClient getClient(Context context) {
+    if (sClient == null) {
+      sClient = new OkHttpClient.Builder().cache(getCache(context)).build();
+    }
+    return sClient;
+  }
+
+  private static Cache getCache(Context context) {
+    int cacheSize = 50 * 1024 * 1024; // 50 MiB
```

Comment:
Where did this number come from?

The largest manifest is *much* smaller than 50:

<img width="760" alt="Screen Shot 2021-02-18 at 10 31 18 AM" src="https://user-images.githubusercontent.com/1220444/108404210-72ebe600-71d4-11eb-9fc8-84c0f1b9f5ab.png">

Would 10 MiB be sufficient or am I being miserly?

Reply:
50 MiB is the recommended size from OkHttp: https://square.github.io/okhttp/caching/ . (Note this is the max cache size, not the immediate size on disk.) I think it needs to be sufficiently larger than an individual response for that response to be cached. We could make this smaller if we're really concerned about it, but since it's in the application's cacheDir anyway, the OS will be able to prune it if it decides it needs more disk space.

Reply:
make sense. Thanks!